run_csm.py

```diff
@@ -0,0 +1,141 @@
+            "for the exam we had recently to be fair that was a more of a last minute scenario "
+            "but like yeah I'm trying to like yeah I noticed this yesterday that like Mondays I "
+            "sort of start the day with this not like a panic but like a"
+        ),
+        "audio": "prompts/conversational_a.wav",
+        "hf_path": "spaces/sesame/csm-1b/prompts/conversational_a.wav"
+    },
+    "conversational_b": {
+        "text": (
+            "like a super Mario level. Like it's very like high detail. And like, once you get "
+            "into the park, it just like, everything looks like a computer game and they have all "
+            "these, like, you know, if, if there's like a, you know, like in a Mario game, they "
+            "will have like a question block. And if you like, you know, punch it, a coin will "
+            "come out. So like everyone, when they come into the park, they get like this little "
+            "bracelet and then you can go punching question blocks around."
+        ),
+        "audio": "prompts/conversational_b.wav",
+        "hf_path": "spaces/sesame/csm-1b/prompts/conversational_b.wav"
+    }
+}
+
+def ensure_prompts_downloaded():
+    """Download prompt files if they don't exist locally"""
+    os.makedirs("prompts", exist_ok=True)
+
+    for prompt_info in SPEAKER_PROMPTS.values():
+        local_path = prompt_info["audio"]
+        if not os.path.exists(local_path):
+            try:
+                hf_hub_download(
+                    repo_id="sesame/csm-1b",
+                    filename=prompt_info["hf_path"].split("/")[-1],
+                    repo_type="space",
+                    local_dir="prompts"
+                )
+                print(f"Downloaded {local_path}")
+            except Exception as e:
+                print(f"Error downloading {local_path}: {str(e)}")
+                raise
+
+def load_prompt_audio(audio_path: str, target_sample_rate: int) -> torch.Tensor:
+    audio_tensor, sample_rate = torchaudio.load(audio_path)
+    audio_tensor = audio_tensor.squeeze(0)
+    if sample_rate != target_sample_rate:
+        audio_tensor = torchaudio.functional.resample(
+            audio_tensor, orig_freq=sample_rate, new_freq=target_sample_rate
+        )
+    return audio_tensor
+
+def prepare_prompt(text: str, speaker: int, audio_path: str, sample_rate: int) -> Segment:
+    audio_tensor = load_prompt_audio(audio_path, sample_rate)
+    return Segment(text=text, speaker=speaker, audio=audio_tensor)
+
+def main():
+    # Download prompts if needed
+    ensure_prompts_downloaded()
+
+    # Select the best available device, skipping MPS due to float64 limitations
+    if torch.cuda.is_available():
+        device = "cuda"
+    else:
+        device = "cpu"
+    print(f"Using device: {device}")
+    
+    try:
+        # Load model
+        model_path = hf_hub_download(repo_id="sesame/csm-1b", filename="ckpt.pt")
+        generator = load_csm_1b(model_path, device)
```

Comment:
Can be replaced with simpler loading that's been merged
```suggestion
        generator = load_csm_1b(device)
```